src/Components/ConfirmDialog/ConfirmDialog.styles.ts

```diff
@@ -0,0 +1,27 @@
+import {
+    IConfirmDialogStyleProps,
+    IConfirmDialogStyles
+} from './ConfirmDialog.types';
+import { CardboardClassNamePrefix } from '../../Models/Constants/Constants';
+
+const classPrefix = `${CardboardClassNamePrefix}-confirmdialog`;
+const classNames = {
+    root: `${classPrefix}-root`
+};
+
+// export const CONFIRMDIALOG_CLASS_NAMES = classNames;
```

Comment:
I remember this from another PR as well, what was the reason for keeping these? I thought we had considered not leaving them commented out but using the actual different export name.

Reply:
the reason I don't change the name itself from 'classNames' is because then everywhere you use that internally to this file it's way longer. I could remove from the template though if you feel like it's just bloat.

Reply:
Removed from here though.